### PR TITLE
PRIME-2322 LPN TOA

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

File: prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const COMMUNITY_PHARMACY_ORGANIZATION_AGREEMENT = 'Community Pharmacy Organizati
 const PHARMACY_OBO_TOA = 'Pharmacy On Behalf Of User';
 const DEVICE_PROVIDER_ORG_AGREEMENT = 'Device Provider Organization';
 const PHARMACY_TECHNICIAN_TOA = 'Pharmacy Technician Toa';
+const LICENCED_PRACTICAL_NURSE_TOA = 'Licenced Practical Nurse';
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts:9]: For clarity, should that be `Licenced Practical Nurse Toa`

Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/enums/agreement-type.enum.ts]: it is fixed.

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/AgreementVersionConfiguration.cs [hunk 2]
```diff
@@ -45,6 +45,8 @@ public override IEnumerable<AgreementVersion> SeedData
                     new AgreementVersion { Id = 25, AgreementType = AgreementType.CommunityPracticeOrgAgreement, Text =  "com-practice-org-v2.html",            EffectiveDate = Nov17th2022AgreementUpdate, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
                     new AgreementVersion { Id = 26, AgreementType = AgreementType.CommunityPharmacyOrgAgreement, Text =  "com-pharmacy-org-v2.html",            EffectiveDate = Nov17th2022AgreementUpdate, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
                     new AgreementVersion { Id = 27, AgreementType = AgreementType.DeviceProviderOrgAgreement,    Text =  "device-provider-org-v2.html",         EffectiveDate = Nov17th2022AgreementUpdate, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+
+                    new AgreementVersion { Id = 28, AgreementType = AgreementType.LicencedPracticalNurseTOA,     Text =  "lpn-toa-v1.html",                     EffectiveDate = DateTimeOffset.Now, CreatedTimeStamp = DateTimeOffset.Now, UpdatedTimeStamp = DateTimeOffset.Now },
```
Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/AgreementVersionConfiguration.cs:49]: Make it effective Sept 27, 2022?  
image

Comment (neophyte57) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/AgreementVersionConfiguration.cs:49]: Use `SEEDING_DATE` for `CreatedTimeStamp` and `UpdatedTimeStamp`

Comment (bergomi02) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/AgreementVersionConfiguration.cs]: They are fixed.

**Comment by neophyte57** [comment]
Not sure where problems lies but not able to download LPN TOA post-signing

**Comment by bergomi02** [comment]
> Not sure where problems lies but not able to download LPN TOA post-signing

Local environment is working. Looking into PR to see what is happening there.

**Comment by bergomi02** [comment]
> Not sure where problems lies but not able to download LPN TOA post-signing

I can download TOA with Test card 001.
image

File: prime-dotnet-webapi/Engines/AgreementEngine Internal/CertificationDigest.cs [hunk 3]
```diff
@@ -47,7 +47,14 @@ public static ICertificationDigest Create(ICollection<CertificationDto> certs)
             }
             else
             {
-                return new OtherCollege(regulated, cert);
+                if (License.isLicensedPracticalNurse(cert.License.CurrentLicenseDetail))
```
[the remark this answers is not in the document]
Comment (bergomi02) [reply prime-dotnet-webapi/Engines/AgreementEngine Internal/CertificationDigest.cs:50]: logic is updated according to the comment in the ticket.

File: prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html
Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html:1]: This link won't go to the correct location  
image

Comment (bergomi02) [reply prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html:1]: This should be fixed. FYI, PR DB has been rebuilt.

Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html:1]: Also, please see https://bcgovmoh.atlassian.net/browse/PRIME-2322?focusedCommentId=14978

Comment (bergomi02) [reply prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html:1]: The highlighting has been applied.

File: prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html [hunk 4]
```diff
@@ -0,0 +1,535 @@
...
+      health
+      care providers in British Columbia to a central data system. Every prescription dispensed in community pharmacies
+      is
+      entered into PharmaNet.
+    </p>
+    <p>
+      The purpose of providing you with access to PharmaNet is to enhance patient care by providing timely and relevant
+      information to persons involved in the provision of direct patient care.
+    </p>
+    <p class="bold underline">
```
Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html:24]: Just "bold" please

Comment (bergomi02) [reply prime-dotnet-webapi/Configuration/Database/Resources/lpn-toa-v1.html]: Updated

**Review by neophyte57 — APPROVED** [review]
Thank you!  🙏

**Pull request merged**